packages/flutter_tools/lib/src/build_system/targets/web.dart

```diff
@@ -51,21 +52,29 @@ class WebEntrypointTarget extends Target {
     final String targetFile = environment.defines[kTargetFile];
     final bool shouldInitializePlatform = environment.defines[kInitializePlatform] == 'true';
     final bool hasPlugins = environment.defines[kHasWebPlugins] == 'true';
-    final String import = fs.file(fs.path.absolute(targetFile)).uri.toString();
+    final String importPath = fs.path.absolute(targetFile);
+    final PackageUriMapper packageUriMapper = PackageUriMapper(
```

Comment:
https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#if-you-have-a-newline-after-some-opening-punctuation-match-it-on-the-closing-punctuation

Reply:
Fixed